### DISCUSSION TITLE
FSR on the Client side

### DISCUSSION THREAD

**Discussion by ThiagoRibas-dev** [alvr-org/ALVR#1601] [General]
I have seen some discussion about using AMD's FSR upscaling on VR games, but in those cases the upscaling happens in the computer and the final image is streamed to the headset at the final resolution, correct?

Would it be possible to use FSR upscaling on the client side?
My thinking is that doing so would allow the computer to render the game at a lower resolution and stream the image at a lower resolution and a higher bitrate, which could result in a better final image, lower bandwidth requirement, and maybe even some latency gains, right?

I have no idea if the final image that the headset receives is suitable for that kind of processing, but I figured I might as well ask.

**Comment by zmerp** [comment]
The current generation of headsets is not powerful enough to run FSR, i think. There is a demo for running a mobile-optimized version of FSR on an iphone 12 but it still can't upscale to the high res VR usually requires.
